mllib/src/main/scala/org/apache/spark/ml/feature/HashingTF.scala

```diff
@@ -31,12 +31,9 @@ import org.apache.spark.sql.types.{ArrayType, StructType}
 /**
  * :: Experimental ::
  * Maps a sequence of terms to their term frequencies using the hashing trick.
- * Currently we support two hash algorithms: "murmur3" (default) and "native".
+ * Currently we support one hash algorithms "murmur3" which is also the default option.
```

Comment:
total minor nit s/hash algorithms/hash algorithm/ since it isn't plural anymore.